src/components/Tabs/Tabs.scss

```diff
@@ -244,3 +291,63 @@ $focus-state-box-shadow-color: rgba(92, 106, 196, 0.8);
   visibility: hidden;
   height: 0;
 }
+
+// stylelint-disable selector-max-class, selector-max-combinators, selector-max-specificity
+.newDesignLanguage {
+  // spacing adjustments to account for in measuring
+  &.Tabs {
+    > li:first-of-type {
+      .Tab {
+        padding-left: spacing(tight);
+      }
+    }
+    > li:nth-last-child(2) {
+      .Tab {
+        padding-right: spacing(tight);
+      }
+    }
+  }
```

Comment:
We never know if the activator is visible or not so I'm adding the extra padding to the last Tab, and then removing it from the left of the disclosure.